python/paddle/fluid/layers/nn.py

```diff
@@ -2349,6 +2350,12 @@ def batch_norm(input,
         moving_variance_name(string, Default None): The name of the moving_variance which store the global Variance.
         do_model_average_for_mean_and_var(bool, Default False): Do model average for mean and variance or not.
         fuse_with_relu (bool): if True, this OP performs relu after batch norm.
+        use_global_stats(bool, Default False): Whether to use global mean and
```

Comment:
what does it mean by "global"? All BN use the same mean and variance? Or the BN use the moving mean and variable?